camel-core/src/main/java/org/apache/camel/model/dataformat/FhirJsonDataFormat.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.model.dataformat;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.XmlTransient;
+import org.apache.camel.CamelContext;
+import org.apache.camel.model.DataFormatDefinition;
+import org.apache.camel.spi.DataFormat;
+import org.apache.camel.spi.Metadata;
+
+/**
+ * The FHIR JSON data format is used to marshall/unmarshall to/from FHIR objects to/from JSON.
+ */
+@Metadata(firstVersion = "2.21.0", label = "dataformat,transformation,json", title = "FHIR JSON")
+@XmlRootElement(name = "fhir-json")
```

Comment:
We dont use dash in those xml names, and btw it should be the same as you define in DataFormatsDefinition, so it should be `fhirJson` etc and ditto for the XML guy

Reply:
Is this not for HL7, if so add "hl7" to the label as well